SPECS/policycoreutils/policycoreutils.spec

```diff
@@ -0,0 +1,5344 @@
+mkdir -p %{buildroot}%{_mandir}/man5
+mkdir -p %{buildroot}%{_mandir}/man8
+%{__mkdir} -p %{buildroot}/%{_usr}/share/doc/%{name}/
+
+make -C policycoreutils LSPP_PRIV=y  DESTDIR="%{buildroot}" SBINDIR="%{_sbindir}" LIBDIR="%{_libdir}" SEMODULE_PATH="/usr/sbin" LIBSEPOLA="%{_libdir}/libsepol.a" install
+
+make -C python PYTHON=%{__python3} DESTDIR="%{buildroot}" SBINDIR="%{_sbindir}" LIBDIR="%{_libdir}" LIBSEPOLA="%{_libdir}/libsepol.a" install
+
+make -C semodule-utils PYTHON=%{__python3} DESTDIR="%{buildroot}" SBINDIR="%{_sbindir}" LIBDIR="%{_libdir}" LIBSEPOLA="%{_libdir}/libsepol.a" install
+
+make -C restorecond PYTHON=%{__python3} DESTDIR="%{buildroot}" SBINDIR="%{_sbindir}" LIBDIR="%{_libdir}" LIBSEPOLA="%{_libdir}/libsepol.a" SYSTEMDDIR="/lib/systemd" install
+
+# Fix perms on newrole so that objcopy can process it
+chmod 0755 %{buildroot}%{_bindir}/newrole
+
+# Systemd
+rm -rf %{buildroot}/%{_sysconfdir}/rc.d/init.d/restorecond
+
+rm -f %{buildroot}/usr/share/man/ru/man8/genhomedircon.8.gz
+rm -f %{buildroot}/usr/share/man/ru/man8/open_init_pty.8*
+rm -f %{buildroot}/usr/share/man/ru/man8/semodule_deps.8.gz
+rm -f %{buildroot}/usr/share/man/man8/open_init_pty.8
+rm -f %{buildroot}/usr/sbin/open_init_pty
+rm -f %{buildroot}/usr/sbin/run_init
+rm -f %{buildroot}/usr/share/man/ru/man8/run_init.8*
+rm -f %{buildroot}/usr/share/man/man8/run_init.8*
+rm -f %{buildroot}/etc/pam.d/run_init*
+
+mkdir   -m 755 -p %{buildroot}/%{generatorsdir}
+mkdir   -m 755 -p %{buildroot}/%{_unitdir}
+install -m 644 -p %{SOURCE6} %{buildroot}/%{_unitdir}/
+install -m 644 -p %{SOURCE7} %{buildroot}/%{_unitdir}/
+install -m 644 -p %{SOURCE8} %{buildroot}/%{_unitdir}/
+install -m 755 -p %{SOURCE9} %{buildroot}/%{generatorsdir}/
+install -m 755 -p %{SOURCE5} %{buildroot}/%{_libexecdir}/selinux/
+
+%package python-utils
+Summary:    SELinux policy core python utilities
+Requires:   policycoreutils-python3 = %{version}-%{release}
+BuildArch:  noarch
+
+%description python-utils
+The policycoreutils-python-utils package contains the management tools use to manage
+an SELinux environment.
+
+%files python-utils
+%license python/copying
+%{_sbindir}/semanage
+%{_bindir}/chcat
+%{_bindir}/audit2allow
+%{_bindir}/audit2why
+%{_mandir}/man1/audit2allow.1*
+%{_mandir}/ru/man1/audit2allow.1*
+%{_mandir}/man1/audit2why.1*
+%{_mandir}/ru/man1/audit2why.1*
+%{_mandir}/man8/chcat.8*
+%{_mandir}/ru/man8/chcat.8*
+%{_mandir}/man8/semanage*.8*
+%{_mandir}/ru/man8/semanage*.8*
+%{_datadir}/bash-completion/completions/semanage
+
+%package python3
+Summary: SELinux policy core python3 interfaces
+Requires:policycoreutils = %{version}-%{release}
+Requires:libsemanage-python3 >= %{libsemanagever} libselinux-python3
+Requires: python3-audit
+Requires: checkpolicy
+Requires: setools-python3 >= 4.1.1
+BuildArch: noarch
+
+%description python3
+The policycoreutils-python3 package contains the interfaces that can be used
+by python 3 in an SELinux environment.
+
+%files python3
```

Comment:
Could you group the `%files` sections together after `%package` and `%description` sections to be consistent with the rest of our specs?

Reply:
fixed